src/client/pps/pps.proto

```diff
@@ -110,6 +106,28 @@ message ParallelismSpec {
   double coefficient = 3;
 }
 
+message ObjectStore {
+  string URL = 1;
+}
+
+message SQLDb {
+  string driver = 1;
+  string URL = 2;
+  repeated string init = 3;
+  repeated string tables = 4;
+}
+
+enum ConnectorType {
+  OBJECT_STORE = 0;
+  SQL_DB = 1;
+}
+
+message Connector {
+    ConnectorType type = 1;
```

Comment:
I know that this pattern is what I used for ParallelismSpec, but I think it may be cleaner for users writing config if you omit ConnectorType and just detect which one of `object_store` and `sql_db` are set. Then the config will just look like:

```
{
   ...
   "output": {
      "sql_db": {
         ...
      }
   }
}
```

I wish I'd done parallelism spec this way